### ISSUE TITLE
Support passing the filename as well as a post object

### ISSUE THREAD

**Issue by ArtskydJ** [TehShrike/noddity-render-static#5]
Right now I have to do this:
```js
var opts = { butler: butler, /*...*/ }
butler.getPost('post.md', function (err, post) {
  render(post, opts, cb)
})
```

I want to be able to do this:
```js
var opts = { butler: butler, /*...*/ }
render('post.md', opts, cb)
```
Since the butler is passed into `render`, it should be possible.

Thoughts?

**Comment by TehShrike** [comment]
My instinct is to not add it because it seems like unnecessary code for a feature that isn't that big a deal, but `if (typeof post === 'string') loadFilename(post, options, cb)` does seem simple enough to be worth doing.

**Comment by TehShrike** [comment]
Published as 1.2.0
